packages/core/test/smoke.test.ts

```diff
@@ -373,10 +377,13 @@ describe("shared can be accessed on both server and client", () => {
     // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-explicit-any
     globalThis.window = {} as any;
 
-    expect(env).toMatchObject({
-      NODE_ENV: "development",
-      FOO_BAR: "foo",
-    });
+    const env = lazyCreateEnv();
+
+    expect(() => env.BAR).toThrowErrorMatchingInlineSnapshot(
```

Comment:
Technically we don't have to check this in this test case, but I figure why not🤷